subscriber.html

```diff
@@ -0,0 +1,37 @@
+<html lang="en">
+<body>
+<h1>Subscribe (and Decrypt) Messages</h1>
+<br><br>
+<label for="subscriptions">Your subscriptions will appear here:</label>
+<textarea id="subscriptions" style="width: 100%" rows="15"></textarea>
+</body>
+
+<!-- Include the latest Ably Library  -->
+<script src="https://cdn.ably.io/lib/ably.min-1.js"></script>
+
+<!-- Instance the Ably library  -->
+<script type="text/javascript">
+    var realtime = new Ably.Realtime('INSERT-YOUR-API-KEY-HERE');
+    var channelName = 'encrypted:messages';
+
+    /* Create options object with cipher(encryption) configuration */
+    var channelOpts = {
+        cipher: {
+            key: [
+                1554178354, 192577227, 772660949, -2115679804,
+                1890741037, 2046217291, 703998164, -644134120
+            ]
```

Comment:
The key here should be either an `ArrayBuffer` (e.g. created using `new Uint32Array([1554178354, ...]).buffer`) or a base64 string (e.g. "W29iamVjdCBBcnJheUJ1ZmZlcl0="). An array of ints may happen to work at the moment due to how ably-js browserside works with buffers right now, but isn't a supported type and isn't guaranteed to work in future.

For a tutorial probably best to use a base64 string, it's easy and works in all browsers.

Reply:
I wasn't able to get a string to work. I've updated the example with the `new Uint32Array()` approach

Reply:
If you could let me know how to construct a string representation of a 128/256 bit key, that would be helpful.

Reply:
I've updated the code to use a base64 string as the cipher key